go/tools/asthelpergen/asthelpergen.go

```diff
@@ -320,7 +320,7 @@ func printableTypeName(t types.Type) string {
 	case *types.Named:
 		return t.Obj().Name()
 	case *types.Basic:
-		return strings.Title(t.Name())
+		return strings.Title(t.Name()) //nolint
```

Comment:
`strings.Title` is deprecated in Go 1.18 because it doesn't take into account whitespace in non-English Unicode ranges. That's... most definitely not a concern for the ASCII-only identifiers we're working with here. :sweat_smile: